docs/runners/local.md

```diff
@@ -14,31 +14,98 @@ In order to use the local runner, you need to have a recent version of [docker-c
 Make sure that you have installed Docker compose on your system. You can find more information 
 about this in the [installation](../guides/installation.md) guide.
 
-```bash
-#### Running a Docker compiled pipeline
 
-```bash
-fondant run local <pipeline_ref>
-```
-
-NOTE: that the pipeline ref is the path to the compiled pipeline spec OR a reference to a fondant pipeline in which case a Docker compiler will compile the pipeline
-to a docker compose specification before running the pipeline.This will start the pipeline and provide logs per component (service).
+=== "Console"
+
+    ```bash
+    fondant run local <pipeline_ref>
+    ```
+
+    If you want to use remote paths (GCS, S3, etc.) you can use the `--auth-gcp`, `--auth-aws` or `--auth-azure`.
+    This will mount your default local cloud credentials to the pipeline. Make sure you are authenticated locally before running the pipeline and
+    that you have the correct permissions to access the `base_path` of the pipeline (read/write/create). 
+    
+    You can also use the `--extra_volumes` argument to mount extra credentials or additional files.
+    This volumes will be mounted to every component/service of the docker-compose spec.
+
+
+=== "Python"
+
+    ```python 
+    from fondant.pipeline.compiler import DockerCompiler
+    from fondant.pipeline.runner import DockerRunner
+    
+    EXTRA_VOLUMES = <str_or_list_of_optional_extra_volumes_to_mount>
+    compiler = DockerCompiler(extra_volumnes=EXTRA_VOLUMES)
+    compiler.compile(pipeline=<pipeline_object>)
+
+    runner = DockerRunner()
+    runner.run(input_spec=<path_to_compiled_spec>)
+    ```
+
+    If you want to use remote paths (GCS, S3, etc.) you can use pass the default local cloud credentials to the pipeline.
+
+    === "GCP"
+    
+        ```python
+        from fondant.pipeline.compiler import DockerCompiler
+        from fondant.pipeline.runner import DockerRunner
+        from fondant.core.schema import CloudCredentialsMount
+        
+        gcp_mounr_dir = CloudCredentialsMount.GCP.value
+        compiler = DockerCompiler(extra_volumnes=gcp_mounr_dir)
```

Comment:
```suggestion
        gcp_mount_dir = CloudCredentialsMount.GCP.value
        compiler = DockerCompiler(extra_volumes=gcp_mount_dir)
```

I also think `extra_volumes` needs to be a list.

Same changes in the other tabs.

Reply:
I changed the script to also accept a string (single volume to mount)

Reply:
woops fixed it

Reply:
One more typo remaining :slightly_smiling_face: 

"extra_volumnes" -> "extra_volumes"